spinn_front_end_common/interface/interface_functions/insert_extra_monitor_vertices_to_graphs.py

```diff
@@ -72,35 +72,35 @@ def _handle_second_monitor_functionality(
         for chip in progress.over(machine.chips):
             if not chip.virtual:
                 machine_vertex = self._exists_equiv_vertex(
-                    chip.x, chip.y, machine_graph,
+                    chip.x, chip.y, vertex_to_chip_map,
                     ExtraMonitorSupportMachineVertex)
                 if machine_vertex is None:
                     # add to machine graph
                     machine_vertex = self.__new_mach_monitor(chip)
                     machine_graph.add_vertex(machine_vertex)
 
-                vertex_to_chip_map[chip.x, chip.y] = machine_vertex
-                extra_monitor_vertices.append(machine_vertex)
-
-                # add application graph as needed
-                if application_graph is not None:
-                    equiv_vertex = self._exists_equiv_vertex(
-                        chip.x, chip.y, application_graph, ExtraMonitorSupport)
-                    if equiv_vertex is None:
-                        app_vertex = self.__new_app_monitor(chip)
-                        application_graph.add_vertex(app_vertex)
-                        graph_mapper.add_vertex_mapping(
-                            machine_vertex, Slice(0, 0), app_vertex)
+                    vertex_to_chip_map[chip.x, chip.y] = machine_vertex
+                    extra_monitor_vertices.append(machine_vertex)
+
+                    # add application graph as needed
+                    if application_graph is not None:
+                        equiv_vertex = self._exists_equiv_vertex(
+                            chip.x, chip.y, vertex_to_chip_map,
+                            ExtraMonitorSupport)
```

Comment:
remove as you dont need the type
```suggestion
                            )
```